torch/onnx/utils.py

```diff
@@ -200,6 +200,7 @@ def export(
     keep_initializers_as_inputs: Optional[bool] = None,
     custom_opsets: Optional[Mapping[str, int]] = None,
     export_modules_as_functions: Union[bool, Collection[Type[torch.nn.Module]]] = False,
+    full_model_check: bool = False,
```

Comment:
We'd try avoid adding new arguments in an effort to not make the export api more complicated than it already is. Also a couple of alternatives to consider:

* Always run checker but have it emit warnings instead of errors. Removing the need of this argument.
* Always run checker but have it emit warnings instead of errors based on a new flag. With similar effect like "strict mode", this flag allows suppression of certain errors.

Reply:
Done